eoscompanion/routes.py

```diff
@@ -742,8 +741,7 @@ def _on_got_wrapped_stream(error, result):
             '''
             if error != None:
                 logging.warning(
-                    'Stream wrapping failed {error}'.format(error=error),
-                    file=sys.stderr
+                    'Stream wrapping failed {error}'.format(error=error)
```

Comment:
Question: Where does `logging`'s output go? Does it go to the stdout or stderr? If there's a switch for it to go to the stderr my preference would be to turn that on.